content/news/release-20160508.md

```diff
@@ -0,0 +1,32 @@
+We are proud to announce a new stable release of OpenRA!
+
+This release is the result of 50 different developers, having produced a total of 1,225 commits since this cycle started six months ago. The code base has grown by more than 14,000 lines and almost 1,700 files have been touched during that time.
```

Comment:
The first sentence feels a bit staggered.  How about something like "Development for this release started six months ago, and in this time 50 developers contributed a total of 1,225 commits that added more than 14,000 lines of code."?

I don't think the files touched number is meaningful because of https://github.com/OpenRA/OpenRA/pull/10801.